common/lib/xmodule/xmodule/css/sequence/display.scss

```diff
@@ -429,15 +407,14 @@ nav.sequence-bottom {
   }
 }
 
-.xmodule_VerticalModule div.vert-mod > div ul.sequence-nav-buttons {
-  list-style: none !important;
-}
-
-.xmodule_SequenceModule nav.sequence-bottom ul li.next a,
-.xmodule_SequenceModule nav.sequence-bottom ul li.prev a {
+#seq_content {
 
-/*  &:focus {
-    outline: 0;
+  &:focus,
+  &:active {
+    outline: none;
   }
-*/
+}
+
+.xmodule_VerticalModule div.vert-mod > div ul.sequence-nav-buttons {
+  list-style: none !important;
```

Comment:
Nothing is *that* !important.  Tom Giannattasio did this in 2012: https://github.com/edx/edx-platform/commit/2c827c46f9fd1f2baee0d9408d34bd80533fdedb